server/apps/context.go

```diff
@@ -0,0 +1,115 @@
+package apps
+
+import (
+	"github.com/mattermost/mattermost-plugin-apps/server/store"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+type Context struct {
+	AppID        store.AppID       `json:"-"`
```

Comment:
If `AppID` is ``json:"-"`` then the frontend will not be supplying it in the context. Should we change it so it can be taken from the frontend this way? Or should we instead have the `AppID` as part of the URL or part of the incoming `Wish`?

Reply:
That's an interesting point, I was not considering the client apps, yet.

We may want to choose communicating the AppID differently client->server->plugin, as a path element for instance. There is no point sending the AppID to the App itself - it knows it. Kinda silly/embarrassing/wasteful.